src/app/clusters/groups-server/groups-server.cpp

```diff
@@ -217,22 +195,38 @@ bool emberAfGroupsClusterViewGroupCallback(app::CommandHandler * commandObj, con
     auto groupId     = commandData.groupId;
     auto endpointId  = commandPath.mEndpointId;
 
+    GroupDataProvider::GroupMapping mapping;
     EmberAfStatus status = EMBER_ZCL_STATUS_NOT_FOUND;
     CHIP_ERROR err       = CHIP_NO_ERROR;
-    CharSpan groupName;
+    size_t nameSize      = 0;
 
     if (emberAfCurrentCommand()->type != EMBER_INCOMING_UNICAST && emberAfCurrentCommand()->type != EMBER_INCOMING_UNICAST_REPLY)
     {
         return true;
     }
 
-    if (!IsFabricGroupId(groupId))
+    if (IsFabricGroupId(groupId))
     {
-        status = EMBER_ZCL_STATUS_INVALID_VALUE;
+        GroupDataProvider * provider = GetGroupDataProvider();
+        VerifyOrReturnError(nullptr != provider, false);
```

Comment:
What error occurs to the caller if false is returned here?

Reply:
No respond sent. I have change this part to send a default response with status FAILURE.